ncutixavier/cdld-prada-timelime
# Implement double component

### What does this PR do?
This PR was about to implement double component

### Description of Task to be completed?

- Implement double component

### How should this be manually tested?

- Clone the [repo](https://github.com/ncutixavier/cdld-prada-timelime)
- Checkout branch `feature/add-double-component`
- Run app `npm run serve` and open `http://localhost:8080/#/double-component` in the browser
- To check the code
   - check `double_component` folder under the components folder
   - Change configuration in `doubleComponentConfig.js` under `configs` folder

### What are the relevant pivotal tracker stories?
N/A

### Screenshots
N/A
